app/utils/SecureGCMCipher.scala

```diff
@@ -46,7 +46,7 @@ class SecureGCMCipher @Inject()(implicit private val appConfig: AppConfig) exten
 
   val IV_SIZE = 96
   val TAG_BIT_LENGTH = 128
-  val ALGORITHM_TO_TRANSFORM_STRING = "AES/GCM/PKCS5Padding"
+  val ALGORITHM_TO_TRANSFORM_STRING = "AES/GCM/NoPadding"
```

Comment:
Updated this as found in a slack post it was recommended to fix the errors in the tests https://hmrcdigital.slack.com/archives/C0GS60DK2/p1733923750290349